instat/dlgSummaryBarOrPieChart.vb

```diff
@@ -174,18 +175,20 @@ Public Class dlgSummaryBarOrPieChart
     End Sub
 
     Private Sub SetDialogOptions()
-        If rdoBarChart.Checked Then
-            ucrSaveSummaryBar.SetPrefix("Bar")
-            cmdPieChartOptions.Visible = False
-            cmdBarChartOptions.Visible = True
-            clsRgeomBarFunction.RemoveParameterByName("width")
-            clsBaseOperator.RemoveParameter(clsRCoordPolarParam)
-        ElseIf rdoPieChart.Checked Then
-            ucrSaveSummaryBar.SetPrefix("Pie")
-            cmdPieChartOptions.Visible = True
-            cmdBarChartOptions.Visible = False
-            clsRgeomBarFunction.AddParameter("width", "1")
-            clsBaseOperator.AddParameter(clsRCoordPolarParam)
+        If ucrSaveSummaryBar.bUserTyped = False Then

```

Comment:
Added this IF statement. Before, if the user changed the name to xyz, and then changed tabs - the name would change back to Bar1 or Pie1. But, the user may not want to lose their name! So, now if the user changes their name to xyz and changes tab - it remains as xyz.